specification/privatedns/resource-manager/Microsoft.Network/stable/2020-06-01/privatedns.json

```diff
@@ -0,0 +1,1924 @@
+          "description": "The SOA record in the record set."
+        },
+        "srvRecords": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/SrvRecord"
+          },
+          "description": "The list of SRV records in the record set."
+        },
+        "txtRecords": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/TxtRecord"
+          },
+          "description": "The list of TXT records in the record set."
+        }
+      },
+      "description": "Represents the properties of the records in the record set."
+    },
+    "RecordSet": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The ID of the record set.",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "The name of the record set.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "The type of the record set.",
+          "readOnly": true
+        },
+        "etag": {
+          "type": "string",
+          "description": "The ETag of the record set."
+        },
+        "properties": {
+          "$ref": "#/definitions/RecordSetProperties",
+          "x-ms-client-flatten": true,
+          "description": "The properties of the record set."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Describes a DNS record set (a collection of DNS records with the same name and type) in a Private DNS zone."
+    },
+    "RecordSetListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RecordSet"
+          },
+          "description": "Information about the record sets in the response."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The continuation token for the next page of results.",
+          "readOnly": true
+        }
+      },
+      "description": "The response to a record set list operation."
+    },
+    "Resource": {
+      "description": "The core properties of ARM resources",
+      "properties": {
+        "id": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Fully qualified resource Id for the resource. Example - '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/privateDnsZones/{privateDnsZoneName}'."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The name of the resource"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The type of the resource. Example - 'Microsoft.Network/privateDnsZones'."
+        }
+      },
+      "x-ms-azure-resource": true
+    },
```

Comment:
Sorry, I missed from the previous review.
As of API version 2020-05-01, resources must begin adding the common systemData object property to the top-level property set on read responses (responses for PUT, GET, and PATCH). https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc027-systemdata-support

You can reference https://github.com/Azure/azure-rest-api-specs/blob/20283a103ef5216f805fccc7ace6b6fc26c0cfbd/specification/common-types/resource-management/v2/types.json#L445

Reply:
Since the api version is already available in public, I think supporting systemData will require a version change. Keeping ARM sign off.